python/cudf/cudf/_lib/orc.pyx

```diff
@@ -22,6 +25,19 @@ import errno
 import os
 
 
+cdef unique_ptr[cudf_table] make_table_from_columns(columns):
+    """
+    Cython function to create a `cudf_table` from an ordered dict of columns
+    """
+    cdef vector[gdf_column*] c_columns
+
+    for idx, (col_name, col) in enumerate(columns.items()):
+        check_gdf_compatibility(col._column)
+        c_columns.push_back(column_view_from_column(col._column, col_name))
```

Comment:
Does the current implementation of `to_orc` intend to handle `string` columns? If yes, and libucdf is not handling converting categorical to string columns, some special case might be needed here to use `column_view_from_string_columns`

Reply:
Added an explicit path for `column_view_from_string_columns`, to match CSV writer. It may or map not be required (seems like there is some kind of conversion in place in my testing), but at least it's consistent with CSV writer now...

Reply:
Thanks! Interesting that the conversion is note necessary. I might revisit why that is the case (for my curiosity) sometime in the future. Thanks @j-ieong!